src/Backend/Modules/Blog/Resources/config/doctrine.yml

```diff
@@ -0,0 +1,9 @@
+doctrine:
+  orm:
+    mappings:
+      blog:
+        type: annotation
+        is_bundle: false
+        dir: "%kernel.project_dir%/src/Backend/Modules/Blog/Domain"
```

Comment:
"%kernel.project_dir%/src/Backend/Modules/Blog/Domain/Comment"

Reply:
Fixed in b6fcbe9b98bc6b2fdac6862a242122a9b31f67ec